docs/tutorial/sandbox.md

```diff
@@ -0,0 +1,136 @@
+# Process Sandboxing
+
+One key security feature in Chromium is that processes can executed within a sandbox.
+The sandbox limits the harm that untrusted code can cause by limiting access to most
+system resources — sandboxed processes can only freely use CPU cycles and memory.
+In order to perform operations requiring additional privilege, sandboxed processes
+use dedicated communication channels to delegate tasks to more privileged processes.
+
+In Chromium, sandboxing is mainly applied to renderer processes. In addition, certain
+utility processes are also sandboxed, including the audio service, the GPU service,
+and the storage service. On Windows, the `chrome://sandbox` page will show you the exact
+sandboxing policy associated with each process.
+
+See Chromium's [Sandbox design document][sandbox] for more information.
+
+## Sandboxing in Electron
+
+Electron comes with a mixed sandbox environment, having Chromium's sandbox enabled but
+disabling it by default for renderer processes (meaning that it still applies to the
+relevant utility processes).
+
+Unsandboxed renderers are not a problem for desktop applications that only display
+trusted code, but it makes Electron less secure than Chromium for displaying untrusted
+web content. For applications that require more security, sandboxed renderers are
+recommended.
```

Comment:
We should probably add something here along the lines of:

It is important to note that even what you assume is trusted code may be potentially dangerous, with attack vectors varying from XSS, content injection to MITM attacks on a website you load remotely, there are countless ways attackers could theoretically make your trusted code do untrusted things.  We recommend sandboxing your renderers for all usecases even if it feels like an abundance of caution.

_i think that's the stance we want to have_

Reply:
I love this, but the sentence above might require a few words of background? With your addition it comes off a bit "here's what we do by default, which you shouldn't really do".

Reply:
We should enable `sandbox` by default IMO.

Reply:
Tried to address all of your feedback in deb19b1